flatpak/com.adilhanney.saber.metainfo.xml

```diff
@@ -66,7 +66,8 @@
         </screenshot>
     </screenshots>
 
-        <release version="0.13.1" type="development" date="2023-06-18">
+    <releases>
+         <release version="0.13.1" type="development" date="2023-06-18">
```

Comment:
Thanks for spotting this! Just to keep the tree clean, please can you revert this commit and I'll re-commit it to the main branch.

Reply:
Update: recommited in https://github.com/adil192/saber/commit/916db979c5b05221bdcccb340699137af91ed6ac

Reply:
Sure, this commit is most probably of no use then. Deleting this PR.

Reply:
Why did you delete the whole PR?